lib/membrane_rtmp_plugin/rtmp/source/source_bin.ex

```diff
@@ -111,4 +134,47 @@ defmodule Membrane.RTMP.SourceBin do
   def secure_pass_control(socket, source) do
     :ssl.controlling_process(socket, source)
   end
+
+  defp maybe_link_audio_pad(state) when state.demuxer_audio_pad_ref != nil do
+    {[
+       spec:
+         get_child(:demuxer)
+         |> via_out(state.demuxer_audio_pad_ref)
+         |> child(:audio_parser, %Membrane.AAC.Parser{
+           out_encapsulation: :none
+         })
+         |> child(:funnel_audio, Membrane.Funnel, get_if_exists: true)
+     ], state}
+  end
+
+  defp maybe_link_audio_pad(state) do
+    {[], state}
+  end
+
+  defp maybe_link_video_pad(state) when state.demuxer_video_pad_ref != nil do
+    {[
+       spec:
+         get_child(:demuxer)
+         |> via_out(state.demuxer_video_pad_ref)
+         |> child(:video_parser, Membrane.H264.Parser)
+         |> child(:funnel_video, Membrane.Funnel, get_if_exists: true)
+     ], state}
+  end
+
+  defp maybe_link_video_pad(state) do
+    {[], state}
+  end
+
+  defp assert_pad_count!(name, ctx) do
+    count =
+      ctx.pads
+      |> Map.keys()
+      |> Enum.count(fn pad_ref -> Pad.name_by_ref(pad_ref) == name end)
+
+    if count > 1 do
+      raise("Linking more than one #{name} output pad to #{__MODULE__} is not allowed")
```

Comment:
[NIT] If you don't add `inspect()` it will print something like `Elixir.Membrane.RTMP.SourceBin` instead of `Membrane.RTMP.SourceBin`
```suggestion
      raise("Linking more than one #{name} output pad to #{inspect(__MODULE__)} is not allowed")
```